app/views/applications/show.html.erb

```diff
@@ -3,6 +3,9 @@
 <% @pets.each do |pet| %>
   <%= link_to "#{pet.name}", "/pets/#{pet.id}" %></br>
 <% end %>
+<% if @pets.any? %>
+<p>Why You Would Make a Good Owner: <%= @application.pet_description %><p>
+<% end %>
```

Comment:
had to add this since they explained they wanted to separate descriptions: good home AND good owner